### PR TITLE
[EOS-19644] Add s3 sanity tests for different CORTX versions.

### PR DESCRIPTION
**Pull request by pranavPawar07**
Signed-off-by: Pranav Diliprao Pawar <pranav.pawar@seagate.com>

### CODE REVIEW

File: scripts/s3-sanity/s3-sanity-test.sh [hunk 1]
```diff
@@ -210,8 +210,27 @@ fi
 
 echo "*** S3 Sanity tests start ***"
 
+echo "Replication test starts"
+cmd_out=$(ldapsearch -b "o=s3-background-delete-svc,ou=accounts,dc=s3,dc=seagate,dc=com" -x -w $ldappasswd -D "cn=sgiamadmin,dc=seagate,dc=com" -H ldap://"$end_point")
+accounts_before=$(echo $cmd_out | grep accountId | wc -l)
+echo $accounts_before
+for a in 1 2 3 4 5 6 7 8 9 10
```
Comment (abhilekh268) [inline scripts/s3-sanity/s3-sanity-test.sh:217]: optimize for condition.....

File: scripts/s3-sanity/s3-sanity-test.sh [hunk 2]
```diff
@@ -226,6 +245,8 @@ echo "Create Account"
 output=$(s3iamcli createaccount -n SanityAccountToDeleteAfterUse  -e SanityAccountToDeleteAfterUse@sanitybucket.com --ldapuser sgiamadmin --ldappasswd "$ldappasswd" --no-ssl)
 access_key=$(echo -e "$output" | tr ',' '\n' | grep "AccessKeyId" | awk '{print $3}')
 secret_key=$(echo -e "$output" | tr ',' '\n' | grep "SecretKey" | awk '{print $3}')
+echo "access key " $access_key
```
Comment (abhilekh268) [inline scripts/s3-sanity/s3-sanity-test.sh:248]: why we are printing credentials?

File: scripts/s3-sanity/s3-sanity-test.sh (hunk 1 above)
Comment (nileshgovande) [inline scripts/s3-sanity/s3-sanity-test.sh:217]: for i in {1..10}; do echo $i; done

File: scripts/s3-sanity/s3-sanity-test.sh (hunk 2 above)
Comment (pranavPawar07) [reply scripts/s3-sanity/s3-sanity-test.sh:248]: that was a part of debugging. Removing now..

File: scripts/s3-sanity/s3-sanity-test.sh (hunk 1 above)
Comment (pranavPawar07) [reply scripts/s3-sanity/s3-sanity-test.sh]: done

Comment (pranavPawar07) [reply scripts/s3-sanity/s3-sanity-test.sh]: done please check

**Comment by pranavPawar07** [comment]
Pre merge : http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/1366/console
1 node Mini Prov : http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Mini-Provisioning/job/cortx-s3server/1249/console

File: scripts/s3-sanity/s3-sanity-test.sh [hunk 3]
```diff
@@ -210,6 +210,23 @@ fi
 
 echo "*** S3 Sanity tests start ***"
 
+echo "Replication test starts"
+cmd_out=$(ldapsearch -b "o=s3-background-delete-svc,ou=accounts,dc=s3,dc=seagate,dc=com" -x -w $ldappasswd -D "cn=sgiamadmin,dc=seagate,dc=com" -H ldap://"$end_point")
+accounts_before=$(echo $cmd_out | grep accountId | wc -l)
+END=5
```
Comment (nileshgovande) [inline scripts/s3-sanity/s3-sanity-test.sh:216]: safer side - 10

Comment (nileshgovande) [reply scripts/s3-sanity/s3-sanity-test.sh:216]: Address codecy warinings

File: scripts/s3-sanity/s3-sanity-test.sh [hunk 4]
```diff
@@ -210,6 +210,23 @@ fi
...
+for ((a=1;a<=$END;a++))
```
Comment (nileshgovande) [inline scripts/s3-sanity/s3-sanity-test.sh:217]: @bkirunge7 @abhilekh268 
Is it OK to be serialized or we need it concurrent (parallel)?

File: scripts/s3-sanity/s3-sanity-test.sh (hunk 3 above)
Comment (pranavPawar07) [reply scripts/s3-sanity/s3-sanity-test.sh]: done

**Comment by nileshgovande** [comment]
All jobs passing after manual run

**Pull request merged**